<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>X2Mount</class>
 <widget class="QDialog" name="X2Mount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>512</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>512</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>512</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RST Setup</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="X2Form" name="X2Form">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>4</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>9999</width>
       <height>9999</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>56</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButtonOK">
      <property name="geometry">
       <rect>
        <x>261</x>
        <y>448</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="geometry">
       <rect>
        <x>344</x>
        <y>448</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>24</x>
        <y>96</y>
        <width>424</width>
        <height>216</height>
       </rect>
      </property>
      <property name="title">
       <string>Time and Location</string>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>328</x>
         <y>64</y>
         <width>72</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Sync</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>64</y>
         <width>304</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Sync location,  timezone and time from TheSkyX :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>176</y>
         <width>136</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>RST date and time :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="time_date">
       <property name="geometry">
        <rect>
         <x>152</x>
         <y>176</y>
         <width>248</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>dd/mm/yy  -  hh:mm:ss</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>32</x>
         <y>96</y>
         <width>110</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>RST Longitude :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="longitude">
       <property name="geometry">
        <rect>
         <x>152</x>
         <y>96</y>
         <width>176</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>+XXX YY SS.s</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>32</x>
         <y>120</y>
         <width>110</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>RST Latitude :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="latitude">
       <property name="geometry">
        <rect>
         <x>152</x>
         <y>120</y>
         <width>176</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>+XX YY SS.s</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>32</x>
         <y>144</y>
         <width>110</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>RST Time Zone :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="timezone">
       <property name="geometry">
        <rect>
         <x>152</x>
         <y>144</y>
         <width>176</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>GMT +/-XX</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>32</y>
         <width>328</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Automatically sync time and location on connect</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>146</x>
        <y>56</y>
        <width>144</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>© RTI-Zone.org, 2021</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_logo_2">
      <property name="geometry">
       <rect>
        <x>138</x>
        <y>8</y>
        <width>160</width>
        <height>43</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>RainbowAstro.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="X2_PhotoFileName" stdset="0">
       <string>RainbowAstro.png</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>24</x>
        <y>320</y>
        <width>424</width>
        <height>112</height>
       </rect>
      </property>
      <property name="title">
       <string>Mount settings</string>
      </property>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>144</x>
         <y>32</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Home mount</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>72</y>
         <width>96</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Park position :</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>104</x>
         <y>72</y>
         <width>304</width>
         <height>24</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Parking 1 / +00º 00' 00.0&quot; / 270º 00' 00.0&quot;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Parking 2 / +00º 00' 00.0&quot; / 180º 00' 00.0&quot;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Parking 3 / +00º 00' 00.0&quot; / 090º 00' 00.0&quot;</string>
        </property>
       </item>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>X2Form</class>
   <extends>QFrame</extends>
   <header>x2form.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
